bims/models/biological_collection_cluster.py

```diff
@@ -0,0 +1,27 @@
+# coding=utf-8
+"""Boundary model definition.
+"""
+
+from django.contrib.gis.db import models
+from bims.models.boundary import Boundary
+
+
+class BiologicalCollectionCluster(models.Model):
```

Comment:
I think we should remove Biological from the class name and doc strings etc. Clusters might be about abiotic factors too....